pkg/collector/corechecks/cluster/ksm/kubernetes_state.go

```diff
@@ -553,7 +553,7 @@ func (k *KSMCheck) processLabelsAsTags() {
 	for resourceKind, labelsMapper := range k.instance.LabelsAsTags {
 		labels := make([]string, 0, len(labelsMapper))
 		for label, tag := range labelsMapper {
-			label = "label_" + label
+			label = "label_" + regexp.MustCompile(`[\/]|[\.]|[\-]`).ReplaceAllString(label, "_")
```

Comment:
you should compile once the regexp and store it in a `var` in this package.
it can be done in a `func init() {}` method

Reply:
thank you for the feedback, Cedric.  I just pushed up two commits, please let me know if further changes should be made